### PR TITLE
Documentation Updates

### PR DESCRIPTION
**Pull request by helenvholmes**
This PR does the following:

- Adds a pull request template (copied over from `nypl-d8`)
- Adds a `CHANGELOG.md` in the base of the project and the package folders to be used in the future
- Removes merge conflicts in `docs/docs.md` and fleshes out sections on branching and pull requests.

Blocks #26 and #22.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+## **This PR does the following:**
+-
+
+### UAT Review:
+- [ ] Navigate to [the page in Drupal](https://reno-XXX-nypl.pantheonsite.io/)
+
+### Front End Review:
+- [ ] View [the example in Pattern Lab](https://reno-XXX-nypl.pantheonsite.io/themes/custom/nypl_emulsify/pattern-lab/public)
+- [ ] Compare to [Figma](https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Working-Component-Library?node-id=XXXXXX)
+- [ ] Check against the [Metronome documentation](http://themetronome.co/components/xxx/?fresh=true)
```
Comment (hokei) [inline .github/PULL_REQUEST_TEMPLATE]: Are permissions going to eventually cause problems here?

File: .github/PULL_REQUEST_TEMPLATE [hunk 2]
```diff
@@ -0,0 +1,14 @@
+[Project Reno Jira Ticket](http://jira.nypl.org/browse/RENO-XXX)
+
+[ResearchNow Jira Ticket](http://jira.nypl.org/browse/SFR-XXX)
```
Comment (hokei) [inline .github/PULL_REQUEST_TEMPLATE]: Is this file a WIP?  Lots of reno/drupal specific stuff here

**Comment by hokei** [comment]
A couple of questions, but all in all, looks good

**Review by hokei — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE (hunk 1 above)
Comment (helenvholmes) [reply .github/PULL_REQUEST_TEMPLATE]: Everyone on this repo _should_ have permissions. That said I think that checking the Metronome documentation covers the Patternlab example so I'm going to remove that.

File: .github/PULL_REQUEST_TEMPLATE (hunk 2 above)
Comment (helenvholmes) [reply .github/PULL_REQUEST_TEMPLATE]: I tried to include everything for both projects in this PR template with the idea that users can delete what's not relevant. Do you think there's a better way to do this?

Comment (helenvholmes) [reply .github/PULL_REQUEST_TEMPLATE]: I think actually for now I'm going to recommend we edit this PR template when we come to it. It's not perfect but feel free to open up an issue to discuss improving it if you would like to discuss this more async, Crystal.

**Pull request merged**